meta-client/src/main/java/org/apache/helix/metaclient/api/DataChangeListener.java

```diff
@@ -24,9 +24,9 @@
  */
 public interface DataChangeListener {
   enum ChangeType {
-    NODE_CREATED,     // Node created of the specific path that the listener register to
-    NODE_DELETED,     // Node deleted of the specific path that the listener register to
-    VALUE_UPDATE      // Node value updated
+    ENTRY_CREATED,     // Node created of the specific path that the listener register to
+    ENTRY_DELETED,     // Node deleted of the specific path that the listener register to
+    ENTRY_UPDATE      // Node value updated
```

Comment:
nit, might need to update the comments accordingly, node -> entry and children -> sub entry